bolt/lib/Rewrite/LinuxKernelRewriter.cpp

```diff
@@ -95,7 +95,7 @@ class LinuxKernelRewriter final : public MetadataRewriter {
         return 1;
       if (IP > Other.IP)
         return 0;
-      return ORC == NullORC;
+      return ORC == NullORC && Other.ORC != NullORC;
```

Comment:
Is it important to have an ordering for NullORC entries? Should we order NullORC entries by some sequential id?

Reply:
No need, they are all the same.